src/compiler/utilities.ts

```diff
@@ -478,7 +478,12 @@ namespace ts {
             return getTokenPosOfNode((<SyntaxList>node)._children[0], sourceFile, includeJsDoc);
         }
 
-        return skipTrivia((sourceFile || getSourceFileOfNode(node)).text, node.pos);

```

Comment:
this looks like an OK fix, but I don't think skipTrivia is called on the inside of jsdoc anywhere else. Maybe there should be a completely separate code path?

I haven't thought about it deeply, so it's likely that skipTrivia is 98% right and just needed this tweak to work.

Reply:
Well, it’s definitely true that I didn’t pass the parameter anywhere else. It occurred to me that I could accomplish the same thing by wrapping `skipTrivia` and calling it in a loop with `stopAfterLineBreak`, but I felt like it was a little less clear what was happening. But I can go back to that—I’m pretty ambivalent about it.

Reply:
Ummm, me too. Neither one of these functions are specific to jsdoc, so jsdoc-specific code is surprise in both.

Probably means that this is good enough.